src/dist/component/package.rs

```diff
@@ -349,36 +364,44 @@ fn unpack_without_first_dir<'a, R: Read>(
             continue;
         }
 
-        let size = entry.header().size()?;
-        if size > MAX_FILE_SIZE {
-            // If we cannot tell the user we will either succeed (great), or fail (and we may get a bug report), either
-            // way, we will most likely get reports from users about this, so the possible set of custom builds etc that
-            // don't report are not a great concern.
-            if let Some(notify_handler) = notify_handler {
-                notify_handler(Notification::Error(format!(
-                    "File too big {} {}",
-                    relpath.display(),
-                    size
-                )));
-            }
-        }
-
-        fn flush_ios(
+        /// true if either no sender_entry was provided, or the incremental file
+        /// has been fully dispatched.
```

Comment:
This comment makes no sense to me

Reply:
flush_ios returns true when either:
- sender_entry is None
- entry.read() returns Ok(0)

Rephrasing is welcome

Reply:
Sorry looks like this was me misreading the diff and thinking you'd changed *to* `Result<()>` rather than from it.  Sigh.